code/modules/client/preference_setup/loadout/loadout_utility.dm

```diff
@@ -51,6 +51,18 @@
 	fountainpens["white fountain pen"] = /obj/item/pen/fountain/white
 	gear_tweaks += new/datum/gear_tweak/path(fountainpens)
 
+/datum/gear/utility/hearing_aid
+	display_name = "hearing aid selection"
+	path = /obj/item/device/hearing_aid
+	cost = 1
+
+/datum/gear/utility/hearing_aid/New()
+	..()
+	var/hearingaids = list()
+	hearingaids["hearing aid (skrell design)"] = /obj/item/device/hearing_aid
+	hearingaids["hearing aid (human design)"] = /obj/item/device/hearing_aid/human
```

Comment:
These should probably be "hearing aid, skrell design" and "hearing aid, human design" to keep in line with loadout conventions.